docs/install/iam_policy_us-gov.json

```
@@ -177,6 +177,28 @@
                 "arn:aws-us-gov:elasticloadbalancing:*:*:listener-rule/app/*/*/*"
             ]
         },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "elasticloadbalancing:AddTags"
+            ],
+            "Resource": [
+                "arn:aws:elasticloadbalancing:*:*:targetgroup/*/*",
+                "arn:aws:elasticloadbalancing:*:*:loadbalancer/net/*/*",
+                "arn:aws:elasticloadbalancing:*:*:loadbalancer/app/*/*"
```

```suggestion
                "arn:aws-us-gov:elasticloadbalancing:*:*:targetgroup/*/*",
                "arn:aws-us-gov:elasticloadbalancing:*:*:loadbalancer/net/*/*",
                "arn:aws-us-gov:elasticloadbalancing:*:*:loadbalancer/app/*/*"
```

I'm sure the partition of the `iam_policy_cn.json` also needs updated, but I don't know what it should be.